README.md

```diff
@@ -20,21 +20,21 @@ This is achieved by creating YAML files that contain a pre-determined data schem
 
 Role: [cisco.nac_dc_vxlan.validate](https://github.com/netascode/ansible-dc-vxlan/blob/develop/roles/validate/README.md)
 
-The validate role function is to ensure that the data model is correct, and that the data model is going to be able to be processed by the subsequent roles. The validate role is going to read all the files in the `host_vars` directory and create a single data model in memory for execution.
+The validate role has the function to ensure that the data model is correct, and that the data model is going to be able to be processed by the subsequent roles. The validate role is going to read all the files in the `host_vars` directory that is named after the inventory group defined in your inventory file and create a single data model in memory for execution.
```

Comment:
The documentation in our example repo is more accurate so let's use that language here.